cowprotocol/cow-amm
# Direct control over openzeppelin dependency

Currently, we use a library that comes from a dependency of the composable-cow dependency.
This means that updating composable-cow could lead to also updating openzeppelin, which might not be intended.

This PR only changes our use of libraries: we still preserve interfaces from subdependencies for ease of integration with composable-cow. For now, we try to avoid remappings.

### How to test

Verify that the dependency points to the same commit as the dependency in `composable-cow` (tag `v4.9.3`).
```
$ (cd lib/composable-cow/lib/@openzeppelin/ && git rev-parse HEAD)
fd81a96f01cc42ef1c9a5399364968d0e07e9e90
$ (cd lib/openzeppelin/ && git rev-parse HEAD)
fd81a96f01cc42ef1c9a5399364968d0e07e9e90
```
